modules/jupyter/variables.tf

```diff
@@ -17,6 +17,18 @@ variable "namespace" {
   description = "Kubernetes namespace where resources are deployed"
 }
 
+variable "notebook_image" {
+  type        = string
+  description = "Jupyter notebook image name"
+  default     = "jupyter/tensorflow-notebook"
```

Comment:
In addition to defaulting, can you also explicitly set this value from /applications/jupyter/main.tf so it's clear which app is using which image?

Reply:
Done.